src/Components/App/EmailValidation.js

```diff
@@ -0,0 +1,8 @@
+// 이메일 유효성 체크
+function emailCheck(email) {
+  let regex =
+    /([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/;
+  return email !== "" && email !== "undefined" && regex.test(email);
```

Comment:
조건들에 대해 조금만 더 명시적으로 작성해주면 좋겠어요! 

```suggestion
  if (email === "") { return false }
  if (email === "undefined") { return false }
  return regex.test(email)
```